pages/serverless-containers/reference-content/understanding-serverless-benefits.mdx

```diff
@@ -0,0 +1,91 @@
+---
+meta:
+  title: Understanding Serverless benefits
+  description:
+content:
+  h1: Understanding Serverless benefits
+  paragraph:
+tags: containers serverless
+dates:
+  validation: 2025-05-23
+  posted: 2025-05-23
+categories:
+  - serverless
+  - containers
+---
+
+# What is Serverless Compute?
+
+Serverless computing is a cloud computing model where the cloud providers manages the infrastructure and dynamically alocates computing resources as needed.
+
+This means that you don't have to provision machines, manage a cluster or pay for servers. Your application code is executed on demand and you only pay for the compute time consumed by your software.
+
+This approach enables greater scalability, flexibility and cost-effectiveness.
+
+Key features:
+* **Automatic scaling:** your application can scale up or down automatically based on demand
+* **No server management:** you don't need to provision, maintain or manage servers
+* **Pay as ou go:** you only pay for the compute time you consume, making it cost effective
+
+Serverless is not limited to Serverless Function which is perfect to deploy small chunks of code, it can also directly deploy containers on Serverless Containers and Serverless Jobs.
```

Comment:
```suggestion
Serverless is not limited to Serverless Functions which is perfect to deploy small chunks of code, it can also directly deploy containers on Serverless Containers and Serverless Jobs.
```